ArcGISRuntimeSDKQt_CppSamples/Maps/GenerateOfflineMap/GenerateOfflineMap.cpp

```diff
@@ -99,13 +99,13 @@ void GenerateOfflineMap::generateMapByExtent(double xCorner1, double yCorner1, d
           this, [this](QUuid, const GenerateOfflineMapParameters& params)
   {
     // Take the map offline once the parameters are generated
-    GenerateOfflineMapJob* generateJob = m_offlineMapTask->generateOfflineMap(params, m_tempPath.path() + "/offlinemap.mmpk");
+    GenerateOfflineMapJob* generateJob = m_offlineMapTask->generateOfflineMap(params, m_tempPath.path() + "/offlinemap");
 
     // check if there is a valid job
     if (generateJob)
     {
       // connect to the job's status changed signal
-      connect(generateJob, &GenerateOfflineMapJob::jobStatusChanged, this, [this, generateJob]()
+      connect(generateJob, &GenerateOfflineMapJob::statusChanged, this, [this, generateJob]()
```

Comment:
this new signal actually gives you the `status` as a signal argument. We should use this rather than checking the job property again since it avoids a timing issue when the status has moved on since the signal was fired.

```suggestion
      connect(generateJob, &GenerateOfflineMapJob::statusChanged, this, [this](JobStatus jobStatus)
```

Reply:
@lsmallwood I think we still need to capture `generateJob` right? `JobStatus` only provides access to the status enum. Or should I make `generateJob` a member variable?